manager/controllers/systems_test.go

```diff
@@ -290,3 +290,29 @@ func TestSystemsFilterAdvCount3(t *testing.T) {
 	assert.Equal(t, 1, len(output.Data))
 	assert.Equal(t, 2, output.Data[0].Attributes.RhsaCount)
 }
+
+func TestSystemsSortByOsName(t *testing.T) {
+	utils.SkipWithoutDB(t)
+	core.SetupTestEnvironment()
+
+	w := httptest.NewRecorder()
+	req, _ := http.NewRequest("GET", "/?sort=os_name", nil)
```

Comment:
Could you use `-os_name` for one of these tests? So we will test both variants. Sorry it's the last request to change.